library/globals.inc.php

```diff
@@ -538,6 +527,30 @@ function gblTimeZones()
             '1',
             xl('Display the Donations link on the About page'),
         ],
+    ],
+
+    // Login Page
+    'Login Page' => [
+        'login_page_layout' => array(
```

Comment:
What will happen to upgraders that have center, left, right for this setting. If odd things happen may make sense to change the id of this setting (and then ignore the old setting id).

Reply:
I just noted below that you addressed this in the sql upgrade script, so disregard above post :)